Sources/ProjectSpec/Source.swift

```diff
@@ -41,26 +41,30 @@ extension Source: JSONObjectConvertible {
 
     public init(jsonDictionary: JSONDictionary) throws {
         path = try jsonDictionary.json(atKeyPath: "path")
+
         let maybeCompilerFlagsString: String? = jsonDictionary.json(atKeyPath: "compilerFlags")
         let maybeCompilerFlagsArray: [String]? = jsonDictionary.json(atKeyPath: "compilerFlags")
         compilerFlags = maybeCompilerFlagsArray ??
             maybeCompilerFlagsString.map{ $0.split(separator: " ").map{ String($0) } } ?? []
-        let maybeExcludesString: String? = jsonDictionary.json(atKeyPath: "excludes")
+
         let maybeExcludesArray: [String]? = jsonDictionary.json(atKeyPath: "excludes")
-        excludes = maybeExcludesArray ??
-            maybeExcludesString.map{ $0.split(separator: " ").map{ String($0) } } ?? []        
+        excludes = maybeExcludesArray ?? []
```

Comment:
You can just replace this with `excludes = jsonDictionary.json(atKeyPath: "excludes") ?? []`

Reply:
Done.